tests/functional/ext/consider_ternary_expression/consider_ternary_expression.rc

```diff
@@ -8,3 +8,4 @@ disable=
   unused-variable,
   missing-function-docstring,
   missing-module-docstring,
+  unsupported-assignment-operation,
```

Comment:
In general I dislike this pattern as it makes the test file less explicit. Not sure what others think, but I would take this opportunity to move this list to the test file.

Reply:
Same here, the configuration file should only contains thing that can only be in a conf file like loading a plugin or python interpreters limiting.